plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveFileSystemTestUtils.java

```diff
@@ -142,4 +143,28 @@ public static MaterializedResult filterTable(SchemaTableName tableName, List<Col
             return result.build();
         }
     }
+
+    public static int getSplitsCount(SchemaTableName tableName,
+                                     HiveTransactionManager transactionManager,
+                                     HiveConfig config,
+                                     ConnectorSplitManager splitManager)
+    {
+        ConnectorMetadata metadata = null;
+        ConnectorSession session = null;
+        try (Transaction transaction = newTransaction(transactionManager)) {
+            metadata = transaction.getMetadata();
+            session = newSession(config);
+
+            ConnectorTableHandle table = getTableHandle(metadata, tableName, session);
+
+            metadata.beginQuery(session);
+            ConnectorSplitSource splitSource = getSplits(splitManager, transaction, session, table);
```

Comment:
You need to close it

Reply:
Updated all the occurrences in this utility class